### PR TITLE
Improved scim setup instructions

### PR DESCRIPTION
**Pull request by isemona**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

**Review by edunham — APPROVED** [review]

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 1]
```diff
@@ -380,40 +417,140 @@ passport.use(new BearerStrategy(
...
+// Okta recommended url - https://developer.okta.com/docs/guides/scim-provisioning-integration-prepare/main/#base-url
+app.use('/scim/v2', scimRoute);
+
+const port = process.env.PORT || 3333;
+const server = app.listen(port, () => {
+  console.log(`Listening at http://localhost:${port}/api`);
+});
+server.on('error', console.error);
+```
+### Add HTTP Logging
```
Comment (alisaduncan) [inline _source/_posts/2023-07-28-scim-workshop.md]: I think you need a newline above?

**Review by alisaduncan — COMMENTED** [review]
One comment for a possible newline, but otherwise looks great - very clear instruction changes!

🥟

**Review by alisaduncan — APPROVED** [review]
Oops, meant to approve

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 2]
```diff
@@ -183,18 +183,43 @@ A neat feature with Prisma is the option to view the user table locally. To do t
 
 ## Add a SCIM file and create a scimRoute
 
-For maintainability, we should keep our SCIM implementation code in one place. Let's create a SCIM file, `scim.ts`, in`apps/api/src`, and import it in `main.ts`. As a result, your `scim.ts` file will look like this: 
+For maintainability, we'll aim to keep our SCIM implementation code in one place. Let’s create a SCIM file, scim.ts, in `apps/api/src`, and import it in `main.ts`. At the top of `apps/api/src/scim.ts`, you'll need to import Router to create a scimRoute to export to `apps/api/src/main.ts`. Copy paste the following code to the top of your apps/api/src/scim.ts file:
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md]:
```suggestion
For maintainability, we'll aim to keep our SCIM implementation code in one place. Let's create a SCIM file, scim.ts, in `apps/api/src`, and import it in `main.ts`. At the top of `apps/api/src/scim.ts`, you'll need to import Router to create a scimRoute to export to `apps/api/src/main.ts`. Copy paste the following code to the top of your apps/api/src/scim.ts file:
```

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 3]
```diff
@@ -380,40 +417,140 @@ passport.use(new BearerStrategy(
...
+// '/scim/v2' path appends to every SCIM Endpoints 
+// Okta recommended url - https://developer.okta.com/docs/guides/scim-provisioning-integration-prepare/main/#base-url
+app.use('/scim/v2', scimRoute);
+
+const port = process.env.PORT || 3333;
+const server = app.listen(port, () => {
+  console.log(`Listening at http://localhost:${port}/api`);
+});
+server.on('error', console.error);
+```
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md:488]:
```suggestion
```

```

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 4]
```diff
@@ -380,40 +417,139 @@ passport.use(new BearerStrategy(
...
+// '/scim/v2' path appends to every SCIM Endpoints 
+// Okta recommended url - https://developer.okta.com/docs/guides/scim-provisioning-integration-prepare/main/#base-url
+app.use('/scim/v2', scimRoute);
+
+const port = process.env.PORT || 3333;
+const server = app.listen(port, () => {
+  console.log(`Listening at http://localhost:${port}/api`);
+});
+server.on('error', console.error);
+### Add HTTP Logging
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md]:
```suggestion
```

### Add HTTP Logging
```

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 5]
```diff
@@ -380,40 +417,140 @@ passport.use(new BearerStrategy(
...
+app.use('/scim/v2', scimRoute);
+
+const port = process.env.PORT || 3333;
+const server = app.listen(port, () => {
+  console.log(`Listening at http://localhost:${port}/api`);
+});
+server.on('error', console.error);
+```
+### Add HTTP Logging
+
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md:489]:
```suggestion
```

### Add HTTP Logging
```

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 6]
```diff
@@ -380,40 +417,137 @@ passport.use(new BearerStrategy(
...
+// '/scim/v2' path appends to every SCIM Endpoints 
+// Okta recommended url - https://developer.okta.com/docs/guides/scim-provisioning-integration-prepare/main/#base-url
+app.use('/scim/v2', scimRoute);
+
+const port = process.env.PORT || 3333;
+const server = app.listen(port, () => {
+  console.log(`Listening at http://localhost:${port}/api`);
+});
+server.on('error', console.error);
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md:487]:
```suggestion
server.on('error', console.error);
(```)
### Add HTTP Logging
```

File: _source/_posts/2023-07-28-scim-workshop.md [hunk 7]
```diff
@@ -183,18 +183,43 @@ A neat feature with Prisma is the option to view the user table locally. To do t
...
-For maintainability, we should keep our SCIM implementation code in one place. Let's create a SCIM file, `scim.ts`, in`apps/api/src`, and import it in `main.ts`. As a result, your `scim.ts` file will look like this: 
+For maintainability, we'll aim to keep our SCIM implementation code in one place. Let's create a SCIM file, scim.ts, in `apps/api/src`, and import it in `main.ts`. At the top of `apps/api/src/scim.ts`, you'll need to import Router to create a scimRoute to export to `apps/api/src/main.ts`. Copy paste the following code to the top of your apps/api/src/scim.ts file:
 
 ```ts
 import { Router } from 'express';
 export const scimRoute = Router();
 ```
 
-And you'll import the routes in `main.ts` like this:
+And you’ll then import the scimRoute in `apps/api/src/main.ts` and after `import session from 'express-session';` as shown below:
```
Comment (isemona) [inline _source/_posts/2023-07-28-scim-workshop.md]:
```suggestion
And you'll then import the scimRoute in `apps/api/src/main.ts` and after `import session from 'express-session';` as shown below:
```

**Pull request merged**